HelpSource/lang-sc.js

```diff
@@ -7,11 +7,11 @@ PR.registerLangHandler(
         [PR.PR_STRING,      /^"(?:[^\\"]|\\.)*(?:"|$)/, null, '"'],
     ],
     [
-        /* char literal */
-        [PR.PR_LITERAL,     /^\$(\\)?./],
-        /* symbols */
+        /* symbols, char literals, env vars */
         [PR.PR_ATTRIB_NAME, /^\\\w+/],
         [PR.PR_ATTRIB_NAME, /^'[^']+'/],
+        [PR.PR_ATTRIB_NAME, /^\w+\:/],
+        [PR.PR_ATTRIB_NAME,     /^\$(\\)?./],
```

Comment:
Could you remove the extra white space here?

Reply:
yes, done!

Reply:
You can push a new commit to your remote `topic/docs-syntax-hi` branch and it will be added to this PR. No force pushing required. We have the option of squashing a PR to keep the commit history clean when merging.